src/lib/CheckoutServerApi.ts

```diff
@@ -141,6 +141,7 @@ export default class CheckoutServerApi {
                     currency: Currency.BTC,
                     type: PaymentType.DIRECT,
                     amount: '.0002e8',
+                    vendorMarkup: -.05,
```

Comment:
Is this a different amount than in Demo.ts on purpose? And in the ETH option and the gasPrice?

Reply:
Yes, to test the update functionality.